CoreStubs/ArchiveControl/module.php

```diff
@@ -5,6 +5,7 @@
 class ArchiveControl extends IPSModule
 {
     private $Archive = [];
+    private $AggregatedArchive = [];
```

Comment:
Warum eigentlich. Wir können doch wunderbar Archive verwenden und GetVariableData verwenden und dort nur ein weiteres Feld hinzufügen. Außerdem würde ich dann dort "Data" in "Values" und das neue Feld "AggregatedValues" nennen.